From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Mon, 6 May 2024 19:39:15 -0600
Subject: [PATCH] kernelbase: HACK: Force angle d3d9 for Alpha League.

CW-Bug-Id: #23761
Valve-commit: c7e869c1f61e4c8bbde6bf7c6b22f1bd0dfbbd71
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-steam-game-id.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index 73b6fda..0d4f888 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -530,6 +530,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
         {L"EverQuest 2\\LaunchPad.exe", L" --use-gl=swiftshader"},
         {L"Everquest F2P\\LaunchPad.exe", L" --use-gl=swiftshader"},
         {L"UnrealCEFSubProcess.exe", L" --use-gl=swiftshader", "2316580"},
+        {L"UnrealCEFSubProcess.exe", L" --use-angle=d3d9", "2684500"},
     };
     unsigned int i;
     char sgi[64];
